<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormFindReplace</class>
 <widget class="QWidget" name="FormFindReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>56</height>
   </rect>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelFind">
     <property name="text">
      <string>Find</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="FindLineEdit" name="textToFind">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="buttonFind">
     <property name="icon">
      <iconset theme="edit-find">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="buttonFindPrev_2">
     <property name="icon">
      <iconset theme="dialog-close">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelReplace">
     <property name="text">
      <string>Replace</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="FindLineEdit" name="textToReplace">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="buttonReplace">
     <property name="icon">
      <iconset theme="edit-find-replace">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FindLineEdit</class>
   <extends>QLineEdit</extends>
   <header>findlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonFindPrev_2</sender>
   <signal>clicked()</signal>
   <receiver>FormFindReplace</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
